package.json

```diff
@@ -8,7 +8,6 @@
   "devDependencies": {
     "pulp": "^11.0.0",
     "purescript-psa": "^0.5.0",
-    "purescript": "^0.11.1",
```

Comment:
@garyb same here

Reply:
Yeah, I guess now we have a usable version on npm, may as well 🙂